app/models/sign_in/concerns/certifiable.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SignIn
+  module Concerns
+    module Certifiable
+      extend ActiveSupport::Concern
+
+      def assertion_public_keys
+        @assertion_public_keys ||= certificate_objects.map(&:public_key)
+      end
+
+      def certificate_with_serial(serial)
```

Comment:
I don't see usage of this function either

Reply:
Same as the comment about re: `delete_certificate_with_serial`. Essentially, the job will log out certificate information, including the serial. We can then use that serial to retrieve a copy of the certificate we have loaded or delete if necessary. Both of these methods are convenience methods for us to use via Rails console when responding to an invalid certificate alert.

Reply:
Ohh I see, I might say we remove these for now and then consider functions like this for the eventual 'cert management' project. I don't think we should make functions just for Rails Console use but we might decide to have a couple rake tasks that do this kind of stuff